src/console/dlt-control.c

```diff
@@ -112,6 +113,27 @@ typedef struct {
 } DltReceiveData;
 
 
+void wait_send_buffer_empty(int fd)
+{
+#ifdef __linux__
```

Comment:
This would gives you an error with unused parameter _fd_ when compiling on other PF than Linux.

Reply:
done